main

```diff
@@ -34,12 +37,41 @@ case "$cmd" in
         cp $(jq -r .bvecs config.json) dwi.bvecs
         ;;
 
-    dipy_reconst_dti)
+    dipy_fit_dti)
```

Comment:
As you can see here, we rename some function @soichih. DTI app will not work because there is no `dipy_fit_dti`on DIPY 0.13.0. So this point breaks DTI app

Reply:
OK Thanks!

I just updated to point to 0.14.0 container (doesn't exist yet) and made sure all of the existing BL apps are pointing to 0.13 branch of this repo which still uses dipy_reconst_dti.

Reply:
Great! Thanks!